zzre/tools/ECSExplorer/ECSExplorer.Standard.cs

```diff
@@ -61,6 +61,8 @@ private static void AddStandardEntityNaming()
         AddEntityNamerByComponent<ScrDeck>(High, nameof(ScrDeck));
         AddEntityNamerByComponent<ScrGotCard>(High, nameof(ScrGotCard));
         AddEntityNamerByComponent<ScrRuneMenu>(High, nameof(ScrRuneMenu));
+        AddEntityNamerByComponent<ScrBookMenu>(High, nameof(ScrBookMenu));
+        AddEntityNamerByComponent<ScrMapMenu>(High, nameof(ScrBookMenu));
```

Comment:
`nameof(ScrBookMenu)` copypaste mistake for `ScrMapMenu`

Reply:
thank you, fixed